docs/USAGE-GUIDE.md

```diff
@@ -1548,7 +1554,7 @@ handshake.
 
 **s2n_connection_set_session** de-serializes the session state and updates the connection accordingly.
 
-**s2n_connection_get_session** serializes the session state from connection and copies into the **session** buffer and returns the number of bytes that were copied. If the first byte in **session** is 1, then the next 2 bytes will contain the session ticket length, followed by session ticket and session state. If the first byte in **session** is 0, then the next byte will contain session id length, followed by session id and session state.
+**s2n_connection_get_session** serializes the session state from connection and copies into the **session** buffer and returns the number of bytes that were copied. If the first byte in **session** is 1, then the next 2 bytes will contain the session ticket length, followed by session ticket and session state. If the first byte in **session** is 0, then the next byte will contain session id length, followed by session id and session state. This function was revamped in TLS1.3 to retrieve the last session ticket received by the client.
```

Comment:
Maybe not revamped? That feels like it might go out of date. Let's just describe the behavior.
```suggestion
**s2n_connection_get_session** serializes the session state from connection and copies into the **session** buffer and returns the number of bytes that were copied. If the first byte in **session** is 1, then the next 2 bytes will contain the session ticket length, followed by session ticket and session state. In TLS1.3 (which allows multiple session tickets), the most recent session ticket received will be used. If the first byte in **session** is 0, then the next byte will contain session id length, followed by session id and session state.
```

Reply:
maybe break this into multiple paragraphs and some more info on return type:

```
...number of bytes that were copied. This function can return either a session_id or session_ticket (what determines one over the other?). The first byte in session is used to indicate the return type.

If the first byte i...
```

Reply:
I tried to break it up a little and clarify what is going on.